src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1OutOfServiceBedTest.kt

```diff
@@ -107,8 +107,14 @@ class Cas1OutOfServiceBedTest : InitialiseDatabasePerClassTestBase() {
         .isUnauthorized
     }
 
+    val test = arrayOf("blah")
```

Comment:
is this used?

Reply:
oops! i was trying something out and shouldn't have checked that in. good spot, i've removed it.